### DISCUSSION TITLE
Run-real time false issue

### DISCUSSION THREAD

**Discussion by vmotakat** [GRIDAPPSD/gridappsd-forum#142] [API]
Hi, 
I started simulation offline mode expecting 1 min resolution data by making run-real time false. But my results indicated that simulations running for every 3 sec even with false mode.

<img width="739" alt="Screen Shot 2022-08-26 at 11 08 37 AM" src="https://user-images.githubusercontent.com/102381298/186957041-d72d871c-d99f-4c86-a592-7e9e6f649d76.png">
<img width="270" alt="Screen Shot 2022-08-26 at 11 08 27 AM" src="https://user-images.githubusercontent.com/102381298/186957054-17bd5a5c-2c0a-4314-b26f-b1494f598fc5.png">

In the past I am able to get 1 min resolution but not sure with recent changes.

@poorva1209

**Comment by poorva1209** [comment]
@vmotakat Do you still need this functionality? Just confirming before I merge the pull request.

**Reply by vmotakat** [reply]
I did in my local machine and it working fine. You may add this because it provide flexibility to the user.
